tools/clang/test/CodeGenDXIL/hlsl/types/longvec-scalarized-intrinsics-sm68.hlsl

```diff
@@ -0,0 +1,182 @@
+  // CHECK: fdiv fast float
+  // CHECK: fdiv fast float
+  // CHECK: fsub fast float
+  // CHECK: fsub fast float
+  // CHECK: fsub fast float
+  // CHECK: fsub fast float
+  // CHECK: fcmp fast oge float
+  // CHECK: fcmp fast oge float
+  // CHECK: fcmp fast oge float
+  // CHECK: fcmp fast oge float
+  // CHECK: call float @dx.op.unary.f32(i32 6, float %{{.*}}) ; FAbs(value)
+  // CHECK: call float @dx.op.unary.f32(i32 6, float %{{.*}}) ; FAbs(value)
+  // CHECK: call float @dx.op.unary.f32(i32 6, float %{{.*}}) ; FAbs(value)
+  // CHECK: call float @dx.op.unary.f32(i32 6, float %{{.*}}) ; FAbs(value)
+  // CHECK: call float @dx.op.unary.f32(i32 22, float %{{.*}}) ; Frc(value)
+  // CHECK: call float @dx.op.unary.f32(i32 22, float %{{.*}}) ; Frc(value)
+  // CHECK: call float @dx.op.unary.f32(i32 22, float %{{.*}}) ; Frc(value)
+  // CHECK: call float @dx.op.unary.f32(i32 22, float %{{.*}}) ; Frc(value)
+
+  // CHECK: fsub fast float
+  // CHECK: fsub fast float
+  // CHECK: fsub fast float
+  // CHECK: fsub fast float
+  // CHECK: select i1 %{{.*}}, float %{{.*}}, float
+  // CHECK: select i1 %{{.*}}, float %{{.*}}, float
+  // CHECK: select i1 %{{.*}}, float %{{.*}}, float
+  // CHECK: select i1 %{{.*}}, float %{{.*}}, float
+  // CHECK: fmul fast float
+  // CHECK: fmul fast float
+  // CHECK: fmul fast float
+  // CHECK: fmul fast float
+  res += fmod(vec1, vec3);
+
+  // CHECK: call float @dx.op.unary.f32(i32 21, float %{{.*}}) ; Exp(value)
+  // CHECK: call float @dx.op.unary.f32(i32 21, float %{{.*}}) ; Exp(value)
+  // CHECK: call float @dx.op.unary.f32(i32 21, float %{{.*}}) ; Exp(value)
+  // CHECK: call float @dx.op.unary.f32(i32 21, float %{{.*}}) ; Exp(value)
+  // CHECK: fmul fast float
+  // CHECK: fmul fast float
+  // CHECK: fmul fast float
+  // CHECK: fmul fast float
+  res += ldexp(vec1, vec2);
+
+  // CHECK: call float @dx.op.unary.f32(i32 23, float %{{.*}}) ; Log(value)
+  // CHECK: call float @dx.op.unary.f32(i32 23, float %{{.*}}) ; Log(value)
+  // CHECK: call float @dx.op.unary.f32(i32 23, float %{{.*}}) ; Log(value)
+  // CHECK: call float @dx.op.unary.f32(i32 23, float %{{.*}}) ; Log(value)
+  // CHECK: fmul fast float
+  // CHECK: fmul fast float
+  // CHECK: fmul fast float
+  // CHECK: fmul fast float
+  // CHECK: call float @dx.op.unary.f32(i32 21, float %{{.*}}) ; Exp(value)
+  // CHECK: call float @dx.op.unary.f32(i32 21, float %{{.*}}) ; Exp(value)
+  // CHECK: call float @dx.op.unary.f32(i32 21, float %{{.*}}) ; Exp(value)
+  // CHECK: call float @dx.op.unary.f32(i32 21, float %{{.*}}) ; Exp(value)
+  res += pow(vec1, vec2);
+
+  // CHECK: mul i32
+  // CHECK: call i32 @dx.op.tertiary.i32(i32 49, i32 %{{.*}}, i32 %{{.*}}, i32 %{{.*}}) ; UMad(a,b,c)
+  // CHECK: call i32 @dx.op.tertiary.i32(i32 49, i32 %{{.*}}, i32 %{{.*}}, i32 %{{.*}}) ; UMad(a,b,c)
+  // CHECK: call i32 @dx.op.tertiary.i32(i32 49, i32 %{{.*}}, i32 %{{.*}}, i32 %{{.*}}) ; UMad(a,b,c)
+  res += dot(ivec1, ivec2);
```

Comment:
was going to comment on dot re: https://github.com/microsoft/hlsl-specs/pull/597 but having this test here is good enough! Will make sure its handled when that lowering goes in.